Assets/Mona/TemplateHelper/TemplateHelper.cs

```diff
@@ -0,0 +1,246 @@
+#if UNITY_EDITOR
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Unity.EditorCoroutines.Editor;
+using UnityEngine.Networking;
+
+namespace Mona
+{
+    [InitializeOnLoad]
+    public class TemplateCheck
+    {
+        public static readonly string at_PublicAPIKey = "keywC7DhH4fzXGWcg";
```

Comment:
Same as earlier comment:

"Since these are readonly, can we use a naming convention similar to constants: AIRTABLE_PUBLIC_API_KEY"